content/en/developers/guide/data-collection-resolution.md

```diff
@@ -1,69 +1,70 @@
+| Cloud                               | Google Cloud                                                                      | API crawler                                                                                                                           | 5 min ([default][4])                                                                           | 1 min                 | 
+| Cloud                               | Oracle Cloud Infrastructure                                                                       | Metric collector                                                                                                                           | Real-time                                                                           | 1 min                 | 
+| Cloud Cost Management               | AWS                                                                               | Cost and Usage Report                                                                                                                 | 1 hour                                                                                         | 1 day                 | 
+| Cloud Cost Management               | Azure                                                                             | Cost Exports                                                                                                                          | 1 hour                                                                                         | 1 day                 | 
+| Cloud Cost Management               | Google Cloud                                                                      | Detailed Usage Cost Export                                                                                                            | 1 hour                                                                                         | 1 day                 | 
+| Cloud SIEM                          | Security Signals                                                                  | Datadog Cloud SIEM                                                                                                                    | Real time                                                                                      | 1 millisecond         | 
+| Cloud Security Management           | Findings                                                                          | Datadog Cloud Security Management Misconfigurations                                                                                   | 15 minutes to 4 hours depending on resource type                                               | 1 minute              | 
```

Comment:
If you have some spare bandwidth lying around, I noticed that sometimes we write out `minute` and sometimes we abbreviate it to `min` inconsistently in this table. It definitely predates your edits here, so no pressure to change it; I just thought I'd point it out just in case.

Reply:
good call! updated.